src/plugins/shared_ux/public/components/empty_state/analytics_no_data_page.component.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { KibanaNoDataPage } from '@kbn/shared-ux-components';
+
+interface Props {
+  kibanaGuideDocLink: string;
+  onDataViewCreated: (dataView: unknown) => void;
+}
+
+const solution = i18n.translate('sharedUX.noDataConfig.analytics', {
+  defaultMessage: 'Analytics',
+});
+const pageTitle = i18n.translate('sharedUX.noDataConfig.analyticsPageTitle', {
+  defaultMessage: 'Welcome to Analytics!',
+});
+const addIntegrationsTitle = i18n.translate('sharedUX.noDataConfig.addIntegrationsTitle', {
+  defaultMessage: 'Add integrations',
+});
+const addIntegrationsDescription = i18n.translate(
+  'sharedUX.noDataConfig.addIntegrationsDescription',
+  {
+    defaultMessage: 'Use Elastic Agent to collect data and build out Analytics solutions.',
+  }
+);
+
+export const AnalyticsNoDataPageComponent = ({ kibanaGuideDocLink, onDataViewCreated }: Props) => {
```

Comment:
Do we really need to create a dedicated component for the Analytics or could we expose different `config` object and then the solutions use the generic `<KibanaNoDataPage />` component with those configs?

It seems that by only exposing the recommended `config` object it would still leave the possibility to customize some config if needed (e.g. the logo)

It also seems less verbose if we start to have multiple of those. WDYT?

Reply:
Clint and I talked about having a config, that could be defined by each plugin the start time. We'd have almost like a register of configs, so we could do something like 
```
const { register } = useRegister();
const config = register.get(pluginName);
```
I think this is the next phase of the project. But for now, let's keep it simple. This will solve the problem at hand. What you're saying doesn't really clash with what we have; plugins can still use the generic `<KibanaNoDataPage>` component, which takes `config` as a prop.

Reply:
Ok that could work too.

> that could be defined by each plugin

I thought that "Analytics" encompasses multiple plugins so each individual plugin didn't have to provide the config (in which case they could directly pass it to the generic KibanaNoDataPage. 

Just trying to see how we can lower the number of components/tests/stories we need to maintain. If you are on it with Clint all good 👍